src/core/js/libraries/inview.js

```diff
@@ -392,6 +420,7 @@
             var body = $("body")[0].getBoundingClientRect();
             //make sure to get height and width independently if getBoundingClientRect doesn't return height and width;
             measurements.supplimentDimensions = (body.width === undefined);
+            loop.hasRaf = (window.requestAnimationFrame && window.cancelAnimationFrame);
```

Comment:
😃  i am not using firefox 10 , i was giving example of it 
I mean to say can we write this fallback or doesn't need to write this 
window.requestAnimationFrame = window.requestAnimationFrame
    || window.mozRequestAnimationFrame
    || window.webkitRequestAnimationFrame
    || window.msRequestAnimationFrame